index.bs

```diff
@@ -5476,24 +5466,8 @@ partial dictionary MLOpSupportLimits {
                 1. Let |inputWidth| be |inputShape|[2].
                 1. Let |channels| be |inputShape|[3].
         </dl>
-    1. If |outputSizes| is not given, then:
-        1. Let |outputHeight| be |outputSizes|[0].
-        1. Let |outputWidth| be |outputSizes|[1].
-    1. Otherwise:
-        1. Let |outputSizes| be the result of [=MLGraphBuilder/calculating conv2d output sizes=] given |inputHeight|, |inputWidth|, |windowDimensions|[0], |windowDimensions|[1], |padding|, |strides|, and |dilations|.
-        1. Let |outputHeight| be |outputSizes|[0].
-        1. Let |outputWidth| be |outputSizes|[1].
-        1. Switch on |roundingType|:
-            <dl class=switch>
-                : {{MLRoundingType/"floor"}}
-                ::
-                    1. Set |outputWidth| to floor(|outputWidth|).
-                    1. Set |outputHeight| to floor(|outputHeight|).
-                : {{MLRoundingType/"ceil"}}
-                ::
-                    1. Set |outputWidth| to ceiling(|outputWidth|).
-                    1. Set |outputHeight| to ceiling(|outputHeight|).
-            </dl>
+    1. Let |outputHeight| be |outputSizes|[0].
```

Comment:
FYI, [Infra recently grew new syntax](https://infra.spec.whatwg.org/#example-list-multiple-assignment) to streamline this:

1. Let « _outputHeight_, _outputWidth_ » be _outputSizes_.

No change needed here, I'll do PR for a bulk pass when I'm back at work (soon!)

Reply:
@inexorabletash, I'm so glad to hear you're back soon. 💪

On behalf of the group, welcome back!